src/python_pachyderm/mixin/pfs.py

```diff
@@ -326,17 +328,17 @@ def inspect_commit(
         commit : Union[str, tuple, dict, Commit, pfs_proto.Commit]
             The commit to inspect. Can either be a commit ID or a commit object
             that represents a subcommit (commit at the repo-level).
-        commit_state : pfs_proto.CommitState, optional
+        commit_state : {pfs_proto.CommitState.STARTED, pfs_proto.CommitState.READY, pfs_proto.CommitState.FINISHING, pfs_proto.CommitState.FINISHED}, optional
```

Comment:
Does the `{a, b, c}` syntax help the docs visualize this better?

Reply:
It makes it clear what the possible values are. For other docs, when I see args that can be a limited number of values, the style is typically to list the possible values in a set like so.